src/utils/config.ts

```diff
@@ -26,55 +26,46 @@ export class Config {
     protected defaultDtFormat = "YYYY-MM-DD"
     protected defaultFileExtension = "md"
     protected defaultIndexFile = ".contents"
-    // constructor(targetDir?: string) {
-    //     this.targetPath = path.resolve(HOME, targetDir || ".notes")
-    // }
 
     /**
      * If the configuration for `note-mgr` doesn't already exist, initialize it
      */
-    async init({
+    init({
         defaultDtFormat = this.defaultDtFormat,
         defaultFileExtension = this.defaultFileExtension,
         defaultIndexFile = this.defaultIndexFile,
         targetDir,
     }: ConfigurationOptions) {
         this.targetPath = path.resolve(HOME, targetDir || ".notes")
-        fsPromises.access(this.CONFIG_PATH).catch(() => {
-            const baseConfig = new Map()
-            baseConfig.set(ConfigurationKeys.DEFAULT_DATE_FORMAT, defaultDtFormat)
-            baseConfig.set(ConfigurationKeys.DEFAULT_FILE_EXTENSION, defaultFileExtension)
-            baseConfig.set(ConfigurationKeys.NOTES_ROOT_DIR, this.targetPath)
-            baseConfig.set(ConfigurationKeys.NOTES_INDEX_FILE, defaultIndexFile)
-            this.writeConfig(baseConfig)
-        })
+        const baseConfig = new Map()
+        baseConfig.set(ConfigurationKeys.DEFAULT_DATE_FORMAT, defaultDtFormat)
+        baseConfig.set(
+            ConfigurationKeys.DEFAULT_FILE_EXTENSION,
+            defaultFileExtension
+        )
+        baseConfig.set(ConfigurationKeys.NOTES_ROOT_DIR, this.targetPath)
+        baseConfig.set(ConfigurationKeys.NOTES_INDEX_FILE, defaultIndexFile)
+        this.writeConfig(baseConfig)
```

Comment:
avoids a bug which was only updated the config when there wasn't an existing one. if you re-init, it should work just as before.